errbot/backends/slack.py

```diff
@@ -334,7 +334,15 @@ def _message_event_handler(self, event):
             text = event['text']
             user = event.get('user', event.get('bot_id'))
 
+        mentioned = []
+
+        for word in text.split(' '):
```

Comment:
This will fail on newlines, double spaces, etc. Using `text.split()` would be better as it splits on any form of (consecutive) whitespace.